lib/test/createClientWithFundedWallet.js

```diff
@@ -1,14 +1,7 @@
-const {
-  PrivateKey,
-} = require('@dashevo/dashcore-lib');
-
 const Dash = require('dash');
 
 const getDAPISeeds = require('./getDAPISeeds');
-
-const fundAddress = require('./fundAddress');
-const wait = require('../wait');
-
+const fundAccount = require('./fundAccount');
```

Comment:
We already have `fundWallet` function in Wallet Lib. Let's just reuse it? https://github.com/dashevo/wallet-lib/pull/119/files#diff-1c8a94b728d0cccd2708ade2269dd0c6

Reply:
Yeah, I've saw that, but it's in wallet-lib. 
Here we are dependant on `dash` repo.

Reply:
Yeah, we defo don't want to duplicate this logic everywhere, so I was thinking that we can include wallet-lib (that branch atm) into dependencies and reuse the `fundWallet` function.

Reply:
This is done, we now will need to wait the PR (in WLib)